src/components/furniture/separators/Separator3.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div
+    class="separator"
+    :style="{
+      width: spaceWidth + '%',
+    }"
+  ></div>
+  <div
+    class="separator"
+    :style="{
+      width: spaceWidth + '%',
+    }"
+  ></div>
+  <div
+    class="separator"
+    :style="{
+      width: spaceWidth + '%',
+    }"
+  ></div>
+</template>
+
+<script>
+export default {
+  props: ['spaceWidth'],
+};
+</script>
+
+<style scoped>
+.separator {
+  display: inline-block;
+  height: 100%;
+  border-right: 2px solid rgb(117, 62, 14);
+}
+</style>
```

Comment:
comon, merge this Separetor1, Speparator2 and 3 in 1 single component with a bit more of logic, you can do it ;)